internal/pkg/service/cli/dialog/create_bucket.go

```diff
@@ -0,0 +1,72 @@
+package dialog
+
+import (
+	"github.com/keboola/go-client/pkg/keboola"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/options"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/cli/prompt"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+	"github.com/keboola/keboola-as-code/pkg/lib/operation/project/remote/create/bucket"
+)
+
+type createBucketDeps interface {
+	Options() *options.Options
+}
+
+func (p *Dialogs) AskCreateBucket(d createBucketDeps) (bucket.Options, error) {
+	opts := bucket.Options{}
+
+	if d.Options().IsSet("name") {
+		opts.Name = d.Options().GetString("name")
+	} else {
```

Comment:
To name sa teraz musi zadavat viac krat, prislo by mi jednoduchsie to upravit
- `stage`
- potom na `display name`
- vygeneroval by som default value pre `name` (`strhelper.NormalizeName(display name)`) 
- a na koniec sa opytat na `description`

Reply:
👍 upravil jsem. A přidal raději i interaktivní e2e test.